### PR TITLE
Improved doc2

### PR DESCRIPTION
**Pull request by tomalin**
Improved documentation in README.md and comments in code for WiresLongVM.py

### CODE REVIEW

**Comment by tomalin** [comment]
I should mention that I made two changes to the code itself in WiresLongVM.py .
1) Used separate variable names fp1, fp2, fp3, fp4 for each of the 4 files written instead of a single name fp, to make it easier to identify which file is being accessed.
2) Simplified adding of entries to maps (which happens several times, and each time can be reduced from 9 lines to 2) by using python dictionaries. e.g. https://github.com/cms-tracklet/project_generation_scripts/compare/improvedDoc2#diff-50f4db734f276c1af14759ba271d38abL774 .

**Comment by tomalin** [comment]
I should mention that I made two changes to the code itself in WiresLongVM.py .
1) Used separate variable names fp1, fp2, fp3, fp4 for each of the 4 files written instead of a single name fp, to make it easier to identify which file is being accessed.
2) Simplified adding of entries to maps (which happens several times, and each time can be reduced from 9 lines to 2) by using python dictionaries. e.g. https://github.com/cms-tracklet/project_generation_scripts/compare/improvedDoc2#diff-50f4db734f276c1af14759ba271d38abL774 .

**Comment by pwittich** [comment]
@djcranshaw can you take a look

File: README.md [hunk 1]
```diff
@@ -133,7 +142,7 @@ and written to corresponding SP memory.
 
 validtepair*(...) checks of each VM pair would be consistent with a track of Pt > 2 GeV.
 
-For displaced tracking, function validtedpair*(...) used instead, which checks if the VM pair is consistent with |d0| < 3cm for all possible abs(q/Pt) < 1/2.
+For displaced tracking, function validtedpair*(...) used instead, which requires the VM pair to be consistent with |d0| < 3cm for at least one of abs(q/Pt) = 1/2 or -1/2.
```
Comment (djcranshaw) [inline README.md]: Remove abs() in this line. Everything else looks good.

Comment (tomalin) [reply README.md]: I agree with Derek's comment and have committed the corresponding fix.

**Pull request merged**